### ISSUE TITLE
Inkbunny is broken

### ISSUE THREAD

**Issue by ZutgKcjA** [NO-ob/LoliSnatcher_Droid#216]
Somehow someway Inkbunny has become broken. I think it has been broken since version 2.3.2 or before. I have checked and rechecked the booru settings in the app, and I have also verified that API access is enabled in user settings on the website. The app just says No Data Loaded irrespective of the tags I try to load.

**Comment by NO-ob** [comment]
Is it still broken for you it works for me?

**Comment by ZutgKcjA** [comment]
That's strange as hell. I don't understand why it doesn't work for me. I have deleted the previous Inkbunny profile and tried to re-add the profile as new, but of course now it doesn't let me save the booru configuration since "Test Booru" returns no data. 

Are you maybe using guest access to view Inkbunny? Since I can say for sure that it does not work (No Data Returned) when you provide a username and password (as API key). 

Just testing this now, when you set username to "guest" and leave password field blank, login is succesful and the API returns pictures (no explicit). Trying to login as a registered user returns no data (login fail).

**Comment by NO-ob** [comment]
Yeah i have an account set, does your password contain special characters?

**Comment by ZutgKcjA** [comment]
Yes it does. 

I remember this was an issue before, but it had also been fixed at some point.

**Comment by NO-ob** [comment]
Hi special characters seem to be working fine. could you enable debug settings by tapping version number at the bottom of the settings menu, goto logging, enable booruHandlerInfo,  booruHandlerRawFetched, booruHandlerFetchFailed, then fill in your info in the add booru page and set the boorutype dropdown to Inkbunny.

before clicking test start making a logcat file ```adb logcat | grep flutter > logcat```  if on windows just do ```adb logcat > logcat```, once the test has finished running cancel the logcat creation with ctrl + c. Before posting the file check it in a text editor 

Look for ```session token found:``` and remove your session token from after that also do a ctrl f for your password incase thats in there and remove them

**Comment by ZutgKcjA** [comment]
I've actually never dealt with ADB before, so I don't think I would've been able to easily do this. But I have been able to fix my problem and here is what I did: 

1. Disabled API access in settings, saved
2. Changed my username
3. Changed my password (to a password with no special characters, just in case, even though you said there is no problem)
4. Enabled API access in settings again, saved
5. Created a new Inkbunny profile in the app with the new username and new password, which tested successfully, and saved

Changing the username and password causes one to be logged out on all devices, which might have been part of the reason for the fix? But this is pure speculation and I know almost nothing. Maybe my problem was just some unlikely freak accident that will not be reproducible. 

I would be happy for you to close this issue if you have no more questions, since the issue is now fixed
